src/string-logger.js

```diff
@@ -35,11 +42,22 @@ class StringLoggerInternal implements IStandardLog {
                       "g",
                   )
                 : new RegExp(escapeRegExp(path.sep), "g");
+
         const normalize = (snippet: string): string =>
-            snippet.replace(regex, "/");
-        this._buffer.push(
-            `${"  ".repeat(this._groupIndent)}${args.map(normalize).join("")}`,
-        );
+            snippet.replace(rootDirRegex, "ROOT_DIR").replace(sepRegex, "/");
+
+        const line = `${"  ".repeat(this._groupIndent)}${args
+            .map(normalize)
+            .join("")}`.trimRight();
```

Comment:
Trailing whitespace isn't really relevant and it can be annoying in inline snapshots when saving as eslint/prettier autofixes away empty lines. So, we trim the trailing whitespace

Reply:
I've run into this elsewhere and agree, so annoying.